src/data/nat/choose/basic.lean

```diff
@@ -267,4 +267,21 @@ begin
     apply zero_le }
 end
 
+-- Inequalities about increasing the first argument
+
+lemma choose_le_succ (a c : ℕ) : choose a c ≤ choose a.succ c :=
+by cases c; simp [nat.choose_succ_succ]
+
+lemma choose_le_add (a b c : ℕ) : choose a c ≤ choose (a + b) c :=
+begin
+  induction b with b_n b_ih,
+  { simp, },
+  exact le_trans b_ih (choose_le_succ (a + b_n) c),
+end
+
+lemma choose_le_choose {a b : ℕ} (c : ℕ) (h : a ≤ b) : choose a c ≤ choose b c :=
+(add_tsub_cancel_of_le h) ▸ choose_le_add a (b - a) c
+
+lemma choose_mono (b : ℕ) : monotone (λ a, choose a b) := λ _ _, choose_le_choose b
```

Comment:
I can't seem to add this to @eric-wieser's comments above:

It looks lemmas like the first often are named `*_mono` (cf. [`tsum_mono`](https://leanprover-community.github.io/mathlib_docs/topology/algebra/infinite_sum.html#tsum_mono)), and lemmas like the second are named `monotone_*` (cf. [`monotone_const`](https://leanprover-community.github.io/mathlib_docs/order/monotone.html#monotone_const)). 

On the other hand, there's also [`pow_mono`](https://leanprover-community.github.io/mathlib_docs/algebra/group_power/order.html#pow_mono).

Maybe we need to take a poll on Zulip and add something to the naming conventions doc on this though. Probably we shouldn't let this hold up this PR though.

Reply:
Ok, I do think some more docs and consistency would be good here for sure, it definitely seems both versions are widely used as of right now (even if the second one matches the general guidance better?)

Reply:
I don't think the conventions are particularly well established here; in particular, `tsum_mono` is impossible to state with `monotone`, so it could easily be stated that way just as a best approximation.

Reply:
Let's do that in another Pr.